modules/overrides/user_configurable_overrides.go

```diff
@@ -222,6 +222,13 @@ func (o *userConfigurableOverridesManager) Forwarders(userID string) []string {
 	return o.Interface.Forwarders(userID)
 }
 
+func (o *userConfigurableOverridesManager) CostAttributionDimensions(userID string) []string {
```

Comment:
iiuc, we are only exposing `Dimensions` in user configurable overrides and MaxCardinality can only be set via runtime overrides?

is my understanding correct?

Reply:
Yes that should be the case.  Dimensions are user-configurable, whereas MaxCardinality is still per-tenant, but controlled by the operator via runtime overrides.